components/x-topic-search/src/lib/get-suggestions.js

```diff
@@ -30,13 +50,6 @@ export default (searchTerm, maxSuggestions, apiUrl, followedTopics) => {
 	const dataSrc = addQueryParamToUrl('count', maxSuggestions, apiUrl, false);
 	let url = addQueryParamToUrl('partial', searchTerm.replace(' ', '+'), dataSrc);
```

Comment:
Can use `const` instead of `let`, now `url` isn't being changed.

Reply:
Oh! Thank you!